composer.json

```diff
@@ -18,12 +18,12 @@
     "require": {
         "php": ">=7.0.0",
         "swiftmailer/swiftmailer": "^6.0.1",
-        "symfony/dependency-injection": "~2.7|~3.4|~4.0",
+        "symfony/dependency-injection": "~2.7|~3.3|~4.0",
         "symfony/http-kernel": "~2.7|~3.0|~4.0",
         "symfony/config": "~2.8|~3.0|~4.0"
```

Comment:
what about these then if we start using 3.3 ?

Reply:
Sure, they can be changed as well IMHO.